packages/components/src/__future__/Tag/Tag/_docs/TagMigration.stories.tsx

```diff
@@ -0,0 +1,321 @@
+import React from "react"
+import { Meta, StoryObj } from "@storybook/react"
+import { fn } from "@storybook/test"
+import { Avatar } from "~components/Avatar"
+import {
+  CautionIcon,
+  ExclamationIcon,
+  InformationIcon,
+  SuccessIcon,
+  LiveIcon,
+} from "~components/Icon"
+import { Tag, RemovableTag } from "../../"
+import LegacyStyles from "./TagLegacy.module.scss"
+
+const meta = {
+  title: "Components/Tag/Future/Migration Guide",
```

Comment:
```suggestion
  title: "Components/Tag/Migration Guide",
```

I think? Feels like the guide should be on the top level, but not too fussed.

Reply:
```suggestion
  title: "Components/Tag/Migration to Future",
```